src/sentry/static/sentry/app/views/performance/transactionDetails/quickTrace.tsx

```diff
@@ -125,90 +134,136 @@ function QuickTraceLite({event, trace, location, organization}: QuickTraceLitePr
+          title={t('View the parent transaction in this trace.')}
+        >
+          <EventNode type="white" pad="right" icon={null}>
+            {t('Parent')}
+          </EventNode>
+        </Tooltip>
+      </EventNodeDropdown>
+    );
+    nodes.push(<TraceConnector key="parent-connector" />);
+  }
+
   nodes.push(
     <EventNode key="current-node" type="black">
       {t('This Event')}
     </EventNode>
   );
 
   if (children.length) {
-    const childrenTarget = generateChildrenEventTarget(
+    const seeChildrenText = tn(
+      'View the child transaction of this event.',
+      'View all child transactions of this event.',
+      children.length
+    );
+    const childrenTarget = generateMultiEventsTarget(
       event,
       children,
       organization,
-      location
+      location,
+      'Children'
     );
     nodes.push(<TraceConnector key="children-connector" />);
     nodes.push(
       <EventNodeDropdown
         key="children-node"
         organization={organization}
         location={location}
-        events={children.sort(
-          (a, b) => b['transaction.duration'] - a['transaction.duration']
-        )}
-        seeMoreText={t('See %s children in Discover', children.length)}
+        events={children}
+        seeMoreText={seeChildrenText}
         seeMoreLink={childrenTarget}
       >
         <Tooltip
           position="top"
           containerDisplayMode="inline-flex"
-          title={tn(
-            'View the child transaction of this event.',
-            'View all child transactions of this event.',
-            children.length
-          )}
+          title={seeChildrenText}
         >
           <EventNode type="white" pad="left" icon={null}>
             {tn('%s Child', '%s Children', children.length)}
           </EventNode>
         </Tooltip>
       </EventNodeDropdown>
     );
+  }
 
+  if (descendants?.length) {
+    const seeDescedantsText = tn(
+      'View the descendant transaction of this event.',
+      'View all descendant transactions of this event.',
+      descendants.length
+    );
+    const descendantsTarget = generateMultiEventsTarget(
+      event,
+      descendants,
+      organization,
+      location,
+      'Descendant'
+    );
     nodes.push(<TraceConnector key="descendants-connector" />);
     nodes.push(
       <EventNodeDropdown
         key="descendants-node"
-        comingSoon
         organization={organization}
         location={location}
-        seeMoreText={t('See trace in Discover')}
-        seeMoreLink={traceTarget}
+        events={descendants}
+        seeMoreText={seeDescedantsText}
+        seeMoreLink={descendantsTarget}
       >
         <Tooltip
           position="top"
           containerDisplayMode="inline-flex"
-          title={t('View all transactions in this trace.')}
+          title={seeDescedantsText}
         >
           <EventNode type="white" pad="left" icon={null}>
-            <StyledIconEllipsis size="xs" />
```

Comment:
Weren't we going to show the Ellipsis (along with ancestors) during the inbetween state of quick-trace + full-trace?

eg. right now during loading of `tasks.create_invoice` the quick trace look like this:
<img width="214" alt="image" src="https://user-images.githubusercontent.com/4205004/109574645-5433fc00-7abe-11eb-8216-650a27d4964e.png">

before popping into this:
<img width="249" alt="image" src="https://user-images.githubusercontent.com/4205004/109574659-5c8c3700-7abe-11eb-8d57-6447cd3bb177.png">

Reply:
I didn't not implement that yet because the transition between the two states currently cause things to shift around.

The intermediate state is marked with `null` so this will be addressed in the fix loading states ticket.

Reply:
Ahh makes sense :+1: